plugins/services/src/js/structs/Service.js

```diff
@@ -54,6 +58,27 @@ module.exports = class Service extends Item {
     return ServiceStatus.NA;
   }
 
+  getRegions() {
+    if (!this._regions) {
+      const regionCounts = (this.get("tasks") || []).reduce(
+        (regions, { region }) => {
+          if (region) {
+            regions[region] = regions[region] ? regions[region] + 1 : 1;
+          }
+
+          return regions;
+        },
+        []
```

Comment:
clearly it works, but wouldn't this be more appropriate as an object vs array?

Reply:
Oops, it was meant to be an object actually. Good catch!